api/managementpb/backup/backups.proto

```diff
@@ -195,6 +195,10 @@ message RemoveScheduledBackupResponse {}
 // Backups service handles backup operations to DB.
 service Backups {
   // StartBackup request backup specified service to location.
+  // Could return the Error message in the details containing specific ErrorCode indicating failure reason:
+  // ERROR_CODE_XTRABACKUP_NOT_INSTALLED
+  // ERROR_CODE_INVALID_XTRABACKUP
+  // ERROR_CODE_INCOMPATIBLE_XTRABACKUP
```

Comment:
Please also `description` option same as we have here: https://github.com/percona/pmm/blob/0f316cc1adf997d39e6afc4aad9e75a809a1c8e3/api/managementpb/checks.proto#L95
to make this information visible is swagger.

Reply:
added